geocity/apps/submissions/templates/submissions/emails/submission_classified_for_services.txt

```diff
@@ -1,4 +1,4 @@
-{% load i18n %}{% autoescape off %}{% blocktranslate %}Bonjour{% endblocktranslate %},
+{% load i18n %}{% autoescape off %}{% blocktranslate %}Bonjour {{ name }}{% endblocktranslate %},
```

Comment:
Is the fullname in `name `?

Reply:
Yes, see here: https://github.com/yverdon/geocity/blob/develop/geocity/apps/submissions/services.py#L193